### PR TITLE
#18 Adding Advanced Properties tip

### PR DESCRIPTION
**Pull request by mallowigi**

### CODE REVIEW

**Comment by kurtextrem** [comment]
I'd say keep the text general and not personal (so: "Many don't know that, but it is " -> "It is").
"It is not even a feature of ES6 (ES2015) since it was defined in ES5.1." sounds like it was defined in 5.1 but removed again in ES6. Better would be: "The feature is available since ES5.1" (or maybe just leave this out)

Apart from that, nice tip! :+1:

**Comment by loverajoel** [comment]
@mallowigi hey can you add link to official documentation?

File: README.md [hunk 1]
```diff
@@ -18,6 +18,103 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter.
 
 # Tips list
 
+## #18 - Advanced object properties
+
```
Comment (loverajoel) [inline README.md:23]: add `> date @username`

**Comment by loverajoel** [comment]
Hi, how you can see the jstips repository changed his format and the `master` branch now is `gh-pages`, now are based in _jekyll blog format_. You can see live here => [jstips.com](http://www.jstips.co/)
For that reason you should update your tip with this [instructions](https://github.com/loverajoel/jstips/blob/gh-pages/CONTRIBUTING.md).

Let me know if you need some help or have suggestion. All are welcome!

Thanks

**Comment by loverajoel** [comment]
@mallowigi I like this tips, can you update the format?

**Pull request closed without merging**